bin/cleanup-code-injections.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+rm -f ./tests/example-all-fail/test/HspecFormatter.hs \
+  ./tests/example-empty-file/test/HspecFormatter.hs \
+  ./tests/example-partial-fail/test/HspecFormatter.hs \
+  ./tests/example-success/test/HspecFormatter.hs \
+  ./tests/example-syntax-error/test/HspecFormatter.hs
+  ./tests/example-all-fail/results.json \
+  ./tests/example-empty-file/results.json \
+  ./tests/example-partial-fail/results.json \
+  ./tests/example-success/results.json \
+  ./tests/example-syntax-error/results.json
+
+git checkout tests/example-all-fail/package.yaml tests/example-all-fail/test/Tests.hs \
```

Comment:
There is a small problem with this solution. What if I was to be modifying the `Tests.hs` or `package.yaml` files? E.g. for an upgrade or something. The changes would be reverted. What we usually do is to temporarily store the old files and then restore once the runner has finished.

Reply:
OK, thanks. That sounds way better, I agree.

I will update to follow this approach 👍